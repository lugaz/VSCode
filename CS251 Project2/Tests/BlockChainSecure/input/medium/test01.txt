327 array
fWUBdi 143.564895 4162
khZ6jR4WxSPwX -29.944929 4546
KxQ -138.666047 4703
eK -48.171403 1092
N1fxgIupcE 171.214542 1315
sxji8Dvylj -90.159090 4111
6t0bzgaN -91.288265 4882
kc11w7GuQYD -68.151481 3949
PVESC 2.040633 4642
bgAvGhZ7r -31.677458 2400
xx2 -162.377195 1721
ErAQ2zmf0e -52.988432 4193
u2eV 14.707401 3828
6lL0 58.731354 4266
MdvZHBFFW5 24.830102 2046
TKkRSk09Bfu -19.563667 1111
aar -39.348411 3310
zztJvW -78.869679 1842
mK3Dg8a4 -176.440471 1488
Fu1 -88.434253 2365
Ri1ay -28.152750 4660
9UyIOuKDi5 179.418462 2051
ftBeW -121.387664 4583
ImFd5ygBBG 43.154691 1709
KgXobPYpK -23.459488 2600
Bf9eiIW29lg3a 128.425493 3333
5fxz 30.757425 3652
9qO7Hvy5ZmS -22.997012 2839
DS7SMef -3.644667 2767
4Dor -215.174002 3267
F -53.819674 2107
J 9.327476 4004
UniM9nHQUe 8.871750 1856
Z3hwmEYD7YA -248.254845 4391
nNhAI -60.050879 2970
WQ3eH1a4 16.366185 3462
dSB9kV3jRQTJ -187.931686 1126
5HeEMJcl -80.286449 4894
TLFddnL3NDP -86.721039 1221
Y9NH7UjwNo 86.495524 3090
sJwWZo0NFP0 153.329337 4431
4dt 105.540155 3438
SartX 201.526176 3466
Z1thQcY 134.940425 3697
sy6LW2OO37VQ -15.029974 1643
v01mb 43.524599 4577
3al -212.636780 3131
m0m0O4T 108.994086 4171
FOFqAlr7PtJ 87.563202 3379
hR5JVXAfgho -189.346691 2714
Mqz2z -175.672077 3819
IT -104.135441 2580
qquj 87.875366 2820
N6PHG6R4GVN 42.504219 3161
wiRbn40W 114.550119 2413
X78 86.171720 2475
tXx -32.851818 3931
USNMqiBFjr 8.583021 4808
Z96AKV 42.744959 2208
aqN -175.424960 2666
mCy -52.993739 1276
htHy7FHa3OJRi 53.059491 2042
WZi3R9D 52.895284 1071
Z5P -76.781684 1865
qumfDRZ1opA -105.610366 2222
wwrcsghiL -42.358988 2769
zv -17.168875 3647
RbyuR4Li 8.670833 2155
i -4.187406 1606
NqP -50.380408 1833
N1TFLIxhEh -65.839852 2454
u 46.542085 1382
YFuY32RY 5.965557 4153
QIo0cGddSmZaz -26.886801 4885
cpe 8.727692 1184
3MC3wEjD3Xh -9.451137 1107
CiPz -137.732168 1287
FW6r -2.177777 2736
bAPhmCfvi 76.429235 3083
By -36.233013 4651
4rYHO 251.788839 4809
D -24.516162 1930
9qqLzVf7Zm3fy 53.917055 2173
QJYnT4 2.206093 2816
DspJggF4RK1 -74.764509 1807
QgwHD9m7f 40.696685 3001
E7lAzQoDZL8 20.526170 2554
Y4 83.278749 2502
rys2z 60.671678 2432
Zj -49.308831 4628
W7X2x -111.660425 4240
d4AFCPFf -149.170743 1536
XuEqMgMY 90.218476 3481
Ju 34.242086 4098
rUhUbyqaxRvRi -19.063956 3064
h6pXYaEX 118.765551 3203
ilPpHukDrvZnY 83.030495 4066
9y9 42.566841 4009
mP6wAE 138.605416 3876
QNRAT6 -43.682604 4368
HTiz82 3.368833 3493
PRp 71.300490 1229
tuHjqg -51.913056 3451
b85StLL3MtG -134.607144 2765
dI0Ji6W19O2bx -105.748521 3605
QNLwDySjwVKR4 -19.276004 3922
Q -22.709959 4059
Dxt8T6xJM 63.919438 4277
c1GPWm -29.908117 2747
92IBIyxww 89.223342 3422
eqgOf7 37.369161 1309
vRu7NOFO2v3 -74.830736 2613
nKwNpBywhWmi2 136.888390 1239
JDpNvgQ 22.471973 2910
8 -66.043657 3582
qBVSGeXSzIB 161.434434 1760
yYMy0gLvspKL -84.708808 2314
JT 56.949989 2972
GHQQWVCNEO4 105.044325 1622
J1o0FE 101.839675 3634
uPVfMITtucRXG -174.437600 3535
IBn3A5aSj -37.924141 1511
BSNCrMXwb 77.329082 1599
HT 39.755926 3306
bB8bq0XP0 24.794365 1957
Ujhg3f -217.611886 2727
rSj 138.298623 4199
KUy1em 122.262284 4932
4LqJeVf 173.012384 3146
VBwJk 10.063167 3030
id1RoCWMCgzga -206.507814 3430
0CtvpeXP1FL 128.017238 2277
Oy1awv5W6SK 71.265846 3236
7CasSDsjL -62.888538 1773
P9pu5Jw4QbMzq 51.416768 2168
Y 129.062971 2132
aJFocX -135.553591 4278
fXC 60.407066 3240
Pbvk0 -206.220311 1304
nryc2Dyi 141.652590 4542
E2Haq 5.718292 1508
GAey6nIMpsF -33.642644 2697
yy1bc7IRAjZD 24.686417 4048
RG2hlzcU -58.478347 4286
WubgEl7t2Q 72.002754 2534
R5ej9BQOR5iaS -48.464534 1644
xLGR5p6ryJ -43.825604 1459
4V82uPnc00uCx -84.565254 1617
kpSHhg -116.563714 1058
jCDz 40.316716 2833
n2CSGb 154.077064 1970
SdPA -169.739975 4303
vBXN -53.027651 3690
p -17.867375 3924
xI6dCvsaQ 101.255116 1296
WJBvdKS 137.133614 4812
6otlpRpDQ2C -87.677861 4006
HxRHnbX37 186.308957 4520
bJTmd 7.275626 4204
3Cu8J6X3aYt9 110.348950 1677
lcnIVNCRrN -47.172586 4735
voePf 120.992037 4135
z1V -157.756924 4930
9j0cH -4.422779 1080
NkOUxVGrRW -30.211942 4304
GEh4ajTYtJ 31.308933 3196
3mmZq 12.675137 1535
7G3H5Y7 41.445285 2679
KM -114.606954 3542
P -20.095517 4883
lp3LnKLioD -96.232805 2845
cVvkY5Rkbh -76.346239 4466
Lo -85.531648 2947
RIXCP0uqzU 35.803963 4396
UsuHMzx -152.311807 2598
FUE 14.468626 3172
uJhH2cNmwlf 57.474363 3593
YLHR4TExvev -108.240622 3698
HAfL -66.051232 3893
pSa5iW7GrWZR -10.021180 4553
f 25.991753 4141
9YdfoFUQM1Qc -56.916142 1968
v2IJj -55.779747 1442
j3hgfoL1bV4 -91.992889 2151
cKJW -98.983914 1503
g 4.332575 3565
o3o1XUWgbu2DE -148.713288 4533
oHtaGHrlurgd -0.257870 2640
SSM2P 36.796682 3470
TJ1b 309.276755 3018
Rk41Q2GVYT -25.961157 3214
eCU 54.444914 4010
H8Y 88.783783 1122
Mf5TLESOuj 18.265545 1913
SaTU6UbaXjzh -52.624556 3905
7FZlzWfeVF 11.377409 1345
URMV9 85.645863 1700
LzHs4wR 98.536885 1764
pux8CTC6Itj 16.700199 1551
zFoijuMZIc2K 233.864933 4137
JeQegMpRN9 23.975197 1344
01WLj8R 72.573245 2713
L9I6jM2044T 91.617196 3809
YO312 113.172019 1781
jH8d 127.785003 4825
paPGXmGZpjoi 2.266163 1576
AGR5WkPmbG -76.894640 1016
X6WmoLf -13.626286 3184
vyZRZWUG6 -94.293555 1167
6BwhAhx -36.493662 2253
x929 -43.489345 2041
W 154.907618 4087
d -101.063325 2996
v6hrt 59.210888 3135
l0f5VdZxXhCOZ -24.236253 4813
4pGMV -124.938003 2431
y -20.306763 4495
drBaQTmItkty 23.303033 2440
kNQqdNHrj -77.487945 2526
xy9m -30.900052 3529
brx7hTBOhlTy -80.138007 1077
Ai 99.963469 1254
5MKhRWkCJ7lJ 66.826461 3989
0RiV6kpj -70.697035 3106
mWjedy -98.715654 3531
DLzBJNHGgQ 200.324845 4370
URB -29.088789 2014
rKmvHA 139.574593 1185
G 94.825461 3559
Gxz99czBA -94.010598 4121
uu 3.218321 1257
ug5nBWvKp 94.429904 1206
1n26Jqxepx -108.952069 3374
N7eSdi -38.453247 3262
15be8f4 157.223634 2330
fCLfV -89.476148 4265
WfeCBWsGJ1 148.647071 2861
i2E -20.659830 1235
4HbGj -17.953379 2210
w 11.911495 3317
awdE5WMN 69.906298 3533
uw1daA8B -54.895199 4935
rmfU6AfgXgx -84.343036 4751
wzzhEv4IDa 223.708278 3048
4A0ST 202.360299 4734
6o -57.637746 1672
3Qo40h0ukGdRG 165.407687 1591
m -233.942524 1204
QAxKj4w3kO 127.998615 2065
Oqk7Kg7 93.329361 1741
tljeM -112.794829 2291
apJbCVCvMe98Y 74.467143 3263
kiXhli8 -24.504433 1671
kXGfqrstak 46.614328 1835
n -169.642781 1052
B 41.820989 2351
dYyyo43dMG -20.664028 1436
c 85.995526 2734
eC3b3 200.079839 2085
2d 112.606740 1871
Xa374Dn -79.877029 1004
5 -25.001905 2745
r8zZGdFtypQ9d -19.438128 3498
3PNGr9N -61.029216 4462
ulW 71.171382 2425
3 162.963107 1285
dq4 76.593160 2464
zFl0vw4 21.052379 1638
vQ7m -38.297996 2164
JIB5s -57.836103 4429
ddBpOu2a4UX 79.739840 2574
QTO1ADz6x 52.339246 2528
9wU21bJaO -1.556299 3195
ZVlJ -58.400401 3813
1NMyi8 76.318876 2360
zhB 0.476375 3611
Yj0WU84gOCLe 26.487172 4875
Lkucc8F3 -146.865720 3574
2hRxN -120.660085 2315
Uh7 74.346819 4640
orry2RllD -44.848375 3639
WKje -46.974465 2120
OVB6 30.250243 3766
tzz7AxwPdTY 163.769825 4836
GenBP3DgS0 147.839640 4334
LAVbKSJeyjcE5 -138.194114 3342
3tx8fVp4 122.527374 4860
xYgzhR -65.188944 3556
VZppEB4MzujB4 -124.927725 4712
kmui6Sxz -68.071484 1992
Yx8D8 -53.525496 4275
6qd -70.799390 2272
Of -60.877511 2094
rSug -135.946978 1326
HeWJ7uFV -28.225106 4170
6H5Ujn1 61.722229 2973
5ta9fSsH -42.714020 2645
kJ 52.986384 2199
6Pr4z7ipu3R -7.424854 4347
jkJLZNpUes -209.083715 1679
WSPiqx -102.604664 4910
o1RV -45.235244 4071
gb 41.306656 4539
SKcCFdxq -121.251914 4166
BgL -182.939052 3644
T 5.119149 4878
VP 70.621263 1956
V6juySYMgPZZ -11.938500 1517
fR9U 91.504993 4239
KnFw -18.660542 1645
gcvrgU2xP1 49.159431 4545
s1JUo -100.789285 2618
y5c 32.812985 4664
cYxIg9zlfbSLR 19.904869 3512
eVpVsx9z9BV -18.451213 4716
kQ -62.552524 1046
RyIhMLK100SXl -79.830882 4430
3dKRh07 -2.163988 1902
kx -126.381577 4948
rKOEnvQN4q 190.525184 1269
GEoO9Zq 156.284659 4146
9Ck8GfrCi1F1 -133.660314 3763
X -103.928739 3092
2eMiE -47.043829 1763
7fM0HC2J1 -67.301923 1722
uo2L0 20.189813 2394
Yj2YnSFr -74.163005 1582
getBlock JT
removeEarliestBlock
getEarliestBlock
updateBlock N1EX 74.895955
getEarliestBlock
getBlock o1RV
removeBlock sy6LW2OO37VQ
updateEarliestBlock 54.533549
addBlock MqrueLNx 102.759701 68
removeBlock 92IBIyxww
removeEarliestBlock
updateEarliestBlock -4.811291
getBlock wiRbn40W
addBlock K14 59.559463 992
getEarliestBlock
updateBlock Z96AKV 4.535961
getBlock ug5nBWvKp
addBlock lSMH -78.392146 1948
addBlock UyNb3o -92.059257 1246
getBlock 4rYHO
updateEarliestBlock 53.803235
removeEarliestBlock
addBlock Ewo5FqT6b3ABQ 27.282123 1787
addBlock Uef472RRpMW 100.388967 4779
addBlock 92IBIyxww -63.532436 805
addBlock rtq5GAcGSA 46.207586 2375
getEarliestBlock
removeBlock d4AFCPFf
addBlock oY0xJGtZj8Xml -52.656491 2364
addBlock guGfIZoWmAGp 75.169398 2681
addBlock SZc7351Gv -60.164999 2914
removeEarliestBlock
removeEarliestBlock
updateEarliestBlock 216.782859
removeBlock NqP
getBlock P9pu5Jw4QbMzq
addBlock QF 18.250354 3561
getBlock z1V
removeEarliestBlock
updateEarliestBlock 69.250320
updateEarliestBlock 160.470488
addBlock 4dVsU1 -101.626350 1441
updateEarliestBlock 73.518109
updateEarliestBlock -44.905531
updateEarliestBlock 9.904117
addBlock oZMlDorPC3b -8.414511 2115
removeBlock tzz7AxwPdTY
updateBlock v2IJj 69.031609
removeBlock DS7SMef
addBlock ccehuA5RHnNY 6.986706 3042
getEarliestBlock
getBlock AGR5WkPmbG
getBlock FUE
addBlock qN 43.413558 3858
removeBlock zFoijuMZIc2K
getEarliestBlock
removeBlock m
addBlock uFg 40.490614 4154
updateBlock LAVbKSJeyjcE5 113.872242
addBlock E -35.199171 3000
updateBlock N6PHG6R4GVN 66.658546
addBlock tgFQUMHI 106.896191 4644
removeBlock kx
getBlock hI8Dl1oSrQ3h
getBlock kc11w7GuQYD
removeBlock x929
addBlock QXUmobN 45.118072 4138
addBlock d4AFCPFf 69.934446 4214
updateBlock vRu7NOFO2v3 95.792365
removeBlock Yj2YnSFr
updateEarliestBlock 7.342557
removeEarliestBlock
getEarliestBlock
getEarliestBlock
removeEarliestBlock
addBlock ZJsP58dOsc2m 76.577339 3133
getBlock 9Ck8GfrCi1F1
getEarliestBlock
removeBlock 9wU21bJaO
getBlock xLGR5p6ryJ
updateEarliestBlock 68.153509
getBlock eqgOf7
addBlock P4Pr -132.051668 1619
addBlock M5FR -81.885226 1767
getBlock MqrueLNx
removeBlock KxQ
addBlock AliCH -130.926746 3485
updateEarliestBlock -118.524936
getEarliestBlock
getBlock v2IJj
removeBlock HT
updateEarliestBlock 28.283005
updateBlock ilPpHukDrvZnY 25.294586
addBlock oq 10.043233 2468
addBlock v17EV 52.206594 750
updateBlock ErAQ2zmf0e -263.421154
addBlock QoPxjgglez1 -68.879804 2319
updateEarliestBlock -11.321065
getEarliestBlock
addBlock Ws80txIqJkBT 156.320483 3322
getBlock 92IBIyxww
getEarliestBlock
updateBlock Y -135.544630
removeEarliestBlock
getEarliestBlock
getBlock E2Haq
removeEarliestBlock
addBlock KqdZ0n6 -153.674372 399
removeEarliestBlock
addBlock OcLiOzBcKAIJ -1.031547 3088
addBlock hUw 128.847265 3600
updateBlock kXGfqrstak 97.830263
removeEarliestBlock
getEarliestBlock
addBlock jJ0fB3y6A 64.499593 4290
removeBlock 5
removeBlock VBwJk
updateBlock dq4 -6.000912
removeBlock 15be8f4
updateEarliestBlock 55.557442
removeEarliestBlock
updateBlock KgXobPYpK 124.178910
addBlock hzV -71.454144 1659
getEarliestBlock
getBlock N7eSdi
removeEarliestBlock
addBlock tzz7AxwPdTY -114.265234 4876
updateBlock i2E -36.873326
updateBlock n 175.738103
addBlock V5rc0 -211.815440 3279
addBlock 0mTdZQ -33.519527 3427
updateEarliestBlock 57.046401
getBlock rUhUbyqaxRvRi
getBlock d4AFCPFf
updateEarliestBlock 132.232891
updateEarliestBlock -73.352493
getEarliestBlock
addBlock PKJuLDQh1 -36.204107 513
removeEarliestBlock
updateBlock uo2L0 21.515803
getEarliestBlock
addBlock d89qO 119.638428 1670
removeEarliestBlock
updateEarliestBlock -132.672695
updateEarliestBlock 11.725692
addBlock DoM 58.092711 3044
addBlock GAizPYWpKyUf -160.460661 3868
removeEarliestBlock
updateEarliestBlock -76.187339
addBlock J7LJhT -193.145925 2883
getBlock KUy1em
addBlock CdebADi5tfL -138.095006 3164
addBlock 7 56.041917 2293
addBlock g3Gm 15.580860 3892
getEarliestBlock
removeBlock fCLfV
addBlock NFaT3QW8w 74.394178 4486
getEarliestBlock
addBlock PfZ 141.870288 142
removeBlock CiPz
updateBlock b85StLL3MtG 168.234318
getEarliestBlock
removeEarliestBlock
removeEarliestBlock
addBlock raimDQKV -21.135849 4792
removeBlock ulW
removeEarliestBlock
getBlock JMBnRFER8vO
removeBlock Uef472RRpMW
removeBlock o1RV
updateEarliestBlock 84.412068
updateEarliestBlock -34.751861
removeBlock ilPpHukDrvZnY
removeEarliestBlock
removeBlock TLFddnL3NDP
removeBlock bAPhmCfvi
addBlock BssLp 85.658972 3523
updateBlock eK -137.286324
removeEarliestBlock
getEarliestBlock
removeBlock nryc2Dyi
removeBlock rSj
getEarliestBlock
removeBlock aEIHnTPzLnv
removeEarliestBlock
addBlock ZMmzX74x8z1 -106.359577 1872
addBlock g0bq -29.480085 106
addBlock NqP 5.124592 2688
updateEarliestBlock -171.252409
updateEarliestBlock 206.087830
updateEarliestBlock 112.285795
getBlock EokQ2JFlH
updateBlock ybCap24C 4.758019
removeEarliestBlock
removeBlock LzHs4wR
getEarliestBlock
getBlock d
addBlock lW2YbGd 25.044515 179
addBlock o1RV -126.982372 2100
updateBlock 5fxz -92.717665
updateBlock HeWJ7uFV 129.033855
removeBlock 6BwhAhx
addBlock VBwJk -86.713460 1612
updateBlock yYMy0gLvspKL -75.182178
