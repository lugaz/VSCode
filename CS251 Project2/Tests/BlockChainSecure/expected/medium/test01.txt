331
89
13
197
26
92
208
47
0
242
315
33
129
289
73
216
293
283
97
7
261
5
323
241
298
290
314
135
18
203
55
126
222
185
51
67
303
146
17
65
125
3
156
80
70
206
112
274
187
9
321
280
20
295
167
143
270
43
276
84
281
79
96
180
105
101
8
141
299
226
179
36
59
307
212
23
122
149
218
118
165
14
258
50
311
75
320
39
42
147
64
35
275
27
312
278
326
34
322
145
305
325
192
215
256
111
330
78
178
10
88
172
37
174
262
211
271
198
151
163
264
196
252
286
294
324
202
69
217
52
106
131
168
250
119
46
74
109
85
108
204
245
282
263
224
327
107
205
116
166
150
214
308
240
200
257
4
243
128
188
195
193
104
29
68
99
194
309
115
201
171
58
60
32
230
90
254
210
266
317
137
44
103
31
87
229
136
154
272
259
100
24
285
54
82
284
237
22
148
49
267
223
302
91
186
269
162
253
40
1
265
244
169
30
21
236
235
255
306
142
72
63
11
94
318
248
182
246
56
121
300
62
81
132
157
181
117
130
191
301
158
41
296
199
233
319
297
48
38
189
207
98
161
173
277
231
102
310
66
221
234
61
176
95
77
2
139
175
152
15
113
249
279
76
190
316
83
328
123
124
16
304
268
227
53
144
120
159
114
247
127
19
183
260
291
12
273
164
155
86
292
45
184
133
153
287
213
239
288
232
313
138
220
6
228
209
140
225
110
177
71
25
219
28
57
93
134
170
JT 56.949989 2972
Xa374Dn -79.877029 1004
AGR5WkPmbG -76.89464 1016
AGR5WkPmbG -76.89464 1016
o1RV -45.235244 4071
sy6LW2OO37VQ -15.029974 1643
true 61
92IBIyxww 89.223342 3422
MqrueLNx 102.759701 68
wiRbn40W 114.550119 2413
true 206
K14 59.559463 992
ug5nBWvKp 94.429904 1206
true 329
true 61
4rYHO 251.788839 4809
n -169.642781 1052
true 238
true 88
false
true 160
brx7hTBOhlTy -80.138007 1077
d4AFCPFf -149.170743 1536
true 231
true 275
true 251
eK -48.171403 1092
3MC3wEjD3Xh -9.451137 1107
NqP -50.380408 1833
P9pu5Jw4QbMzq 51.416768 2168
true 63
z1V -157.756924 4930
H8Y 88.783783 1122
true 26
true 122
tzz7AxwPdTY 163.769825 4836
DS7SMef -3.644667 2767
true 203
ug5nBWvKp 94.429904 1206
AGR5WkPmbG 54.533549 4949
FUE 14.468626 3172
true 68
zFoijuMZIc2K 233.864933 4137
ug5nBWvKp 94.429904 1206
m 9.904117 4958
true 179
true 267
true 54
kx -126.381577 4948
null
kc11w7GuQYD -68.151481 3949
x929 -43.489345 2041
true 53
true 180
Yj2YnSFr -74.163005 1582
TLFddnL3NDP -86.721039 1221
PRp 71.30049 1229
PRp 71.30049 1229
PRp 71.30049 1229
true 177
9Ck8GfrCi1F1 -133.660314 3763
i2E -20.65983 1235
9wU21bJaO -1.556299 3195
xLGR5p6ryJ -43.825604 1459
eqgOf7 37.369161 1309
true 244
true 166
null
KxQ -138.666047 4703
true 76
UyNb3o -92.059257 1246
v2IJj 69.031609 4959
HT 39.755926 3306
true 197
true 170
true 170
uu 3.218321 1257
true 65
null
rKOEnvQN4q 190.525184 1269
rKOEnvQN4q 190.525184 1269
mCy -52.993739 1276
E2Haq 5.718292 1508
mCy -52.993739 1276
true 79
KqdZ0n6 -153.674372 399
true 62
true 294
3 162.963107 1285
CiPz -137.732168 1287
true 139
5 -25.001905 2745
VBwJk 10.063167 3030
15be8f4 157.223634 2330
xI6dCvsaQ 101.255116 1296
true 71
Pbvk0 -206.220311 1304
N7eSdi -38.453247 3262
Pbvk0 -206.220311 1304
true 181
true 176
true 38
rUhUbyqaxRvRi -19.063956 3064
d4AFCPFf 69.934446 4214
JeQegMpRN9 23.975197 1344
true 192
PKJuLDQh1 -36.204107 513
JeQegMpRN9 23.975197 1344
true 79
JeQegMpRN9 23.975197 1344
true 257
true 310
4dVsU1 -101.62635 1441
true 26
KUy1em 122.262284 4932
true 106
true 108
true 245
IBn3A5aSj -37.924141 1511
fCLfV -89.476148 4265
true 117
IBn3A5aSj -37.924141 1511
false
CiPz 55.557442 4973
IBn3A5aSj -37.924141 1511
IBn3A5aSj -37.924141 1511
V6juySYMgPZZ -11.9385 1517
true 149
ulW 71.171382 2425
3mmZq 12.675137 1535
null
Uef472RRpMW 100.388967 4779
o1RV -45.235244 4071
ilPpHukDrvZnY 25.294586 4967
3Qo40h0ukGdRG 165.407687 1591
null
bAPhmCfvi 76.429235 3083
true 2
BSNCrMXwb 77.329082 1599
i -4.187406 1606
nryc2Dyi 141.65259 4542
rSj 138.298623 4199
i -4.187406 1606
null
i -4.187406 1606
true 118
true 223
true 226
null
GHQQWVCNEO4 105.044325 1622
LzHs4wR 98.536885 1764
zFl0vw4 21.052379 1638
d -101.063325 2996
true 229
true 184
6BwhAhx -36.493662 2253
true 192
